package/pigpio/Config.in

```diff
@@ -0,0 +1,13 @@
+config BR2_PACKAGE_PIGPIO
+	bool "pigpio"
+  depends on BR2_USE_MMU # fork()
+  depends on BR2_TOOLCHAIN_HAS_THREADS
+  depends on !BR2_STATIC_LIBS
+
```

Comment:
Remove the extra newline here. Sorry to be picky, but the Buildroot devs are super picky on these things.